tests/linting/eslint/rules/no-alphabetical-sort.test.ts

```diff
@@ -339,3 +339,321 @@ ruleTester.run(`A compare function should be provided when using "Array.prototyp
     },
   ],
 });
+
+ruleTester.run(
+  `A compare function should be provided when using "Array.prototype.toSorted()"`,
+  rule,
+  {
+    valid: [
+      {
+        code: `
+      var arrayOfNumbers = [80, 3, 9, 34, 23, 5, 1];
+      arrayOfNumbers.toSorted((n, m) => n - m);
+      `,
+      },
+      {
+        code: `unknownArrayType.toSorted();`,
+      },
+      {
+        code: `
+      function f(a: any[]) {
+        a.toSorted(undefined);
```

Comment:
Could you adapt these examples so they make sense with `toSorted()` not making the change _in place_?

```suggestion
        return a.toSorted(undefined);
```